molecule.go

```diff
@@ -1,4 +1,3 @@
-// Package molecule is a Go library for parsing protobufs in an efficient and zero-allocation manner.
```

Comment:
Is this not how you comment a package?

Reply:
oh you made this `doc.go` thing. Is that a new thing?

Reply:
It's not new, but i've seen it in cases where the package doesn't have an obvious "main" `.go` file, which is the case here, now.